woocommerce/woocommerce-ios
# [Simplify Product Editing] Remove SPE experiment from product details

Closes: #9519


## Description

We're removing the unsuccessful Simplify Product Editing (SPE) experiment from the app. To make it more reasonable for code review, this is divided into three PRs:

* #9600
* #9601
* #9602 (this PR)

The experiment has been remotely disabled already, so there should be no visible change in behavior from `trunk`.

### Changes

* Removes the SPE feature flag completely.
* Removes the visual changes to the product details screen (in `ProductFormViewController`).
* Removes the changes to product type labels (e.g. `Virtual product`, `Physical product`).
* Removes the "Add Options" row/section in `DefaultProductFormTableViewModel`.
* Reverts the changes to the actions available in product details:
   * Removes the bottom sheet action to convert a simple product to a variable product.
   * Shows the Downloadable Files action only if a product is downloadable.
   * Always shows the Reviews row (even if the product has no reviews).
   * Categories row is visible only if a product has categories; otherwise it is in the "Add more details" bottom sheet.
   * Always shows the product type row.

## Testing instructions


1. Build and run the app.
2. Open the Products tab.
3. Select or add a product to open the product details.
4. Confirm the expected rows are visible, as outlined above, and other expected rows appear in the "Add more details" bottom sheet.

## Screenshots

New product details|Existing product details|Add more details
-|-|-
![Simulator Screenshot - iPhone 14 Pro - 2023-05-02 at 16 08 53](https://user-images.githubusercontent.com/8658164/235709015-78dc9b1b-2d01-41a0-92d2-1e01322a72fd.png)|![Simulator Screenshot - iPhone 14 Pro - 2023-05-02 at 16 10 00](https://user-images.githubusercontent.com/8658164/235708620-cff0f629-0d5c-436a-9c98-aee05d58cad9.png)|![Simulator Screenshot - iPhone 14 Pro - 2023-05-02 at 16 10 03](https://user-images.githubusercontent.com/8658164/235708608-f76be4ec-a78b-4b69-af83-55d9fd32b7b9.png)

---
